controller/api/apis.go

```diff
@@ -2882,7 +2882,14 @@ type RESTCrdDlpGroupConfig struct {
 }
 
 type RESTDlpSensorExport struct {
-	Names []string `json:"names"`
+	Names         []string           `json:"names"`
+	ExportOptions *RESTExportOptions `json:"export_options"`
+}
+
+type RESTExportOptions struct {
+	Type            string `json:"type"`
```

Comment:
Please put a comment here & in apis.yaml if "type" field must be "remote" for now

Reply:
This type field no longer exists. Instead, if a remote export request has the `RemoteExportOptions` field configured, it will trigger a remote export.